internal/logout/logout_test.go

```diff
@@ -33,16 +33,40 @@ func TestLogoutCommand(t *testing.T) {
 		assert.Empty(t, saved)
 	})
 
+	t.Run("removes all Supabase CLI credentials", func(t *testing.T) {
+		storeProvider := &credentials.MockProvider{
+			MockStore: map[string]map[string]string{},
+		}
```

Comment:
How about creating this inside `credentials.MockInit()`?

Reply:
Yeah, my guess was that you might want to inject values in the store at setup time in tests.
But I guess you would use Set methods to do so. 👍